server/config/home_page/topics.json

```diff
@@ -47,10 +47,10 @@
   {
     "id": "equity",
     "title": "Equity",
-    "description": "Wield the power of data to create a more just and equitable world.",
+    "description": "Explore Economic Equity, Education Equity, Health Equity, Justice Equity and more.",
```

Comment:
Justice System Equity (Justice Equity is odd)

Reply:
Done.